app/src/main/java/com/kks/nimblesurveyjetpackcompose/network/Api.kt

```diff
@@ -26,4 +27,7 @@ interface Api {
 
     @GET("api/v1/surveys/{surveyId}")
     suspend fun getSurveyDetail(@Path("surveyId") surveyId: String): BaseResponse<SurveyResponse>
+
+    @POST("api/v1/responses")
+    suspend fun submitSurvey(@Body submitSurveyRequest: SubmitSurveyRequest): BaseResponse<SubmitSurveyRequest>
```

Comment:
Is the response for this request the same as the body? Is it not just an empty response with success or failure status? 🤔